cmd/status.go

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright 2020 Mirantis, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+	"strings"
+
+	"github.com/k0sproject/k0s/pkg/build"
+	"github.com/k0sproject/k0s/pkg/install"
+	ps "github.com/mitchellh/go-ps"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	output string
+
+	status    *K0sStatus
+	statusCmd = &cobra.Command{
+		Use:     "status",
+		Short:   "Helper command for get general information about k0s",
+		Example: `The command will return information about system init, PID, k0s role, kubeconfig and similar.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var err error
+
```

Comment:
I think we should check if we're running this as root or not. we're looking for stuff like process list, `/proc/1234/cmdline` which will not be readable without root privileges